README.md

```diff
@@ -33,3 +33,12 @@ Thinking about using the Jenkins-k8s Operator for your next project?
 
 For further details,
 [see the charm's detailed documentation](https://charmhub.io/jenkins-k8s/docs).
+
+## Generating src docs for every commit
+
+Run the following command:
+
+```bash
+echo -e "tox -e src-docs\ngit add src-docs\n" > .git/hooks/pre-commit
+chmod +x .git/hooks/pre-commit
+```
```

Comment:
`pre-commit` is quite good for running tooling that it supports, like black, isort etc. It doesn't work nicely with tox since tox maintains its own dependencies which pre-commit also wants to do. We're pretty deep in tox so pre-commit is not a very good fit for us (maybe things have changed since I last tried)